src/components/WelcomeBack/WalletList.vue

```diff
@@ -94,8 +94,10 @@ export default {
   watch: {
     wallets: {
       deep: true,
-      handler() {
-        this.currentWallet = this.walletOptions[this.lastWalletOpen]
+      handler(prevValue, newValue) {
+        if (prevValue.length !== newValue.length) {
+          this.currentWallet = this.walletOptions[this.lastWalletOpen]
+        }
```

Comment:
Would it be possible to actually resolve `prevValue[this.lastWalletOpen]`, catch some wallet identifier and try to find its index in `newValue`? Or would it be too overkill?

Reply:
Actually, I don't think the watcher is necessary at all, I will remove it!